Makefile

```diff
@@ -67,10 +67,71 @@ build-all:
 aro: check-release generate
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
 
-.PHONY: runlocal-rp
-runlocal-rp:
-	go run -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro rp
+# Target to create docker secrets
+.PHONY: docker-secrets
+docker-secrets: aks.kubeconfig
+	docker secret rm --ignore aks.kubeconfig
+	docker secret create aks.kubeconfig ./aks.kubeconfig
+
+	docker secret rm --ignore proxy-client.key
+	docker secret create proxy-client.key ./secrets/proxy-client.key
+
+	docker secret rm --ignore proxy-client.crt
+	docker secret create proxy-client.crt ./secrets/proxy-client.crt
 
+	docker secret rm --ignore proxy.crt
+	docker secret create proxy.crt ./secrets/proxy.crt
+
+# Target to run the local RP
+.PHONY: runlocal-rp
+runlocal-rp: ci-rp docker-secrets
+	docker run --rm -p 127.0.0.1:8443:8443 \
+		--name aro-rp \
+		-w /app \
+		-e RP_MODE="development" \
+		-e PROXY_HOSTNAME="$(PROXY_HOSTNAME)" \
+		-e DOMAIN_NAME="$(DOMAIN_NAME)" \
+		-e AZURE_RP_CLIENT_ID="$(AZURE_RP_CLIENT_ID)" \
+		-e AZURE_FP_CLIENT_ID="$(AZURE_FP_CLIENT_ID)" \
+		-e AZURE_SUBSCRIPTION_ID="$(AZURE_SUBSCRIPTION_ID)" \
+		-e AZURE_TENANT_ID="$(AZURE_TENANT_ID)" \
+		-e AZURE_RP_CLIENT_SECRET="$(AZURE_RP_CLIENT_SECRET)" \
+		-e LOCATION="$(LOCATION)" \
+		-e RESOURCEGROUP="$(RESOURCEGROUP)" \
+		-e AZURE_ARM_CLIENT_ID="$(AZURE_ARM_CLIENT_ID)" \
+		-e AZURE_FP_SERVICE_PRINCIPAL_ID="$(AZURE_FP_SERVICE_PRINCIPAL_ID)" \
+		-e AZURE_DBTOKEN_CLIENT_ID="$(AZURE_DBTOKEN_CLIENT_ID)" \
+		-e AZURE_PORTAL_CLIENT_ID="$(AZURE_PORTAL_CLIENT_ID)" \
+		-e AZURE_PORTAL_ACCESS_GROUP_IDS="$(AZURE_PORTAL_ACCESS_GROUP_IDS)" \
+		-e AZURE_CLIENT_ID="$(AZURE_CLIENT_ID)" \
+		-e AZURE_SERVICE_PRINCIPAL_ID="$(AZURE_SERVICE_PRINCIPAL_ID)" \
+		-e AZURE_CLIENT_SECRET="$(AZURE_CLIENT_SECRET)" \
+		-e AZURE_GATEWAY_CLIENT_ID="$(AZURE_GATEWAY_CLIENT_ID)" \
+		-e AZURE_GATEWAY_SERVICE_PRINCIPAL_ID="$(AZURE_GATEWAY_SERVICE_PRINCIPAL_ID)" \
+		-e AZURE_GATEWAY_CLIENT_SECRET="$(AZURE_GATEWAY_CLIENT_SECRET)" \
+		-e DATABASE_NAME="$(DATABASE_NAME)" \
+		-e PULL_SECRET="$(PULL_SECRET)" \
```

Comment:
this breaks cluster install on Linux - you'll see the error:

```
level=error msg="invalid character 'a' looking for beginning of object key string" func="backend.(*openShiftClusterBackend).updateAsyncOperation.func1()" file="pkg/backend/openshiftcluster.go:262"
```

Reply:
I haven't had a chance to trace this error to its source, but my cluster creation attempt got stuck on a similar error on MacOS:

```
msg="step [Action pkg/cluster.(*manager).runHiveInstaller] encountered error: invalid character 'a' looking for beginning of object key string"
```

Maybe there are multiple related things to be fixed here?

Reply:
Thanks, cluster creation works for me now!